pkgs/development/python-modules/numpy/default.nix

```diff
@@ -101,19 +101,40 @@ in buildPythonPackage rec {
   enableParallelBuilding = true;
 
   nativeCheckInputs = [
-    pytest
+    pytestCheckHook
     hypothesis
     typing-extensions
   ];
 
   checkPhase = ''
     runHook preCheck
     pushd "$out"
-    ${python.interpreter} -c 'import numpy, sys; sys.exit(numpy.test("fast", verbose=10) is False)'
+    pytestCheckPhase
     popd
     runHook postCheck
   '';
```

Comment:
Perhaps checking out what they do in their [custom test runner](https://github.com/numpy/numpy/blob/a277f6210739c11028f281b8495faf7da298dbef/numpy/_pytesttester.py#L79) in terms of adding arguments to pytest could help.

1. If we use their existing test runner, it looks like we can pass custom pytest args using the `extra_argv` param.
2. If we switch to pytestCheckHook, do we want to run their full test suite or stick to running their fast test suite?

Reply:
Disabling test using the first option would be uglier, `pytestCheckHook` is a better option.

> do we want to run their full test suite or stick to running their fast test suite?

Fast one would likely lead to less breakages which imo is proffered. I'll try to replicate it

Reply:
Done